packages/loot-core/src/server/accounts/sync.js

```diff
@@ -831,7 +830,7 @@ export async function syncNordigenAccount(userId, userKey, id, acctId, bankId) {
         starting_balance_flag: true,
       });
 
-      let result = await reconcileNordigenTransactions(id, transactions);
+      let result = await reconcileNordigenTransactions(id, transactions.all);
```

Comment:
oops? :) 

Also: there are conflicts with `master`. Would you mind taking a look at those too?

Reply:
Should be good to go now